presto-docs/src/main/sphinx/release/release-0.205.rst

```diff
@@ -0,0 +1,72 @@
+=============
+Release 0.205
+=============
+
+General Changes
+---------------
+
+* Fix parsing of row type signatures with spaces and types containing spaces.
+* Fix distributed planning failure for complex queries when using bucketed execution.
+* Fix a bug that triggers checks for ``SELECT`` privileges for tables in ``DELETE`` queries.
+  This bug only affects connectors that implement ``checkCanSelectFromColumns()``.
+* Fix a bug in enforcing column access controls when querying views. This bug
+  only affects connectors that implement the column access control methods.
+* Fix a bug in :func:`ST_ExteriorRing` to make it only work with polygons.
+* Add the ``task.min-drivers-per-task`` and ``task.max-drivers-per-task`` config options.
+  The former specifies the guaranteed minimum number of drivers a task will run concurrently
+  given that it has enough work to do. The latter specifies the maximum number of drivers
+  that can run concurrently for a task.
+* Add ``concurrent-lifespans-per-task`` config property to control the default value
+  of ``concurrent_lifespans_per_task`` session property.
+* Add ``query_max_total_memory`` session property and ``query.max-total-memory`` config
+  property. Queries will be aborted after their total (user + system) memory reservation
+  exceeds this threshold.
+* Improve the error message when access is denied when selecting from a view due to the
+  view owner having insufficient permissions to create the view.
+* Improve stats calculation for outer joins and correlated subqueries.
+* Change ``DELETE FROM <table> WHERE <condition>`` to check that the user has ``SELECT``
+  privileges on the ``WHERE`` condition, which is more in line with the ANSI standard.
+* Reduce memory usage for columns of all ``NULL`` or all non-null values.
+* Change the internal hash function used in  ``approx_distinct``. For a fixed value set,
+  result of ``approx_distinct`` may change in this version compared to the previous
+  version. The standard error of the results should still be within the configured bounds.
+* Improve efficiency and reduce memory usage for scalar correlated subqueries with aggregation.
+* Remove the legacy local scheduler and associated configuration properties.
+* Reject ``FILTER`` clause for ``IF``, ``COALESCE``, and ``NULLIF`` functions.
+* Disallow non-static ``@ScalarFunction``-annotated methods. Non-static SQL function
+  implementations must now be declared in ``@ScalarFunction``-annotated classes.
+* Disallow having multiple public constructors in ``@ScalarFunction`` classes. All non-static
+  implementations of a SQL function will now be associated with a single constructor.
+  This fixes expressivity gap of providing specialized implementations for SQL functions
```

Comment:
"This improves support for providing specialized implementations of SQL functions with generic arguments."